AWESOME.md

```diff
@@ -15,6 +15,7 @@ To get started, we recommend watching [AutoGluon 1.0: Shattering the AutoML Ceil
 
 | Title                                                                                                                                                              | Format   | Location                                                                                                                           | Date       |
 |--------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------------------------|------------|
+| [AutoGluon: AutoML at Your Fingertips](https://suzhoum.github.io/icml-24-autogluon-talk/)                                                                | Expo Talk | [ICML 2024](https://icml.cc/Expo/Conferences/2024/talk%20panel/36234)                                                            | 2024/07/21 |
```

Comment:
The slides should be made public, currently I can't access them.

Fingertip -> Fingertips

Reply:
Same here, I think the link is only accessible to people registered. Would be good to upload it to our team public S3 bucket

Reply:
I updated the permission. Could you please check again?

Reply:
I can see them now, thanks @suzhoum !